### ISSUE TITLE
Search page design

### ISSUE THREAD

**Issue by jwelby** [thepracticaldev/DEV-ios#62]
Using the search button in the navigation goes to a list of users. This was slightly surprising as I was expecting a general search. This left me uncertain what this page would do.

The search field looks like a page heading. Only by tapping it does it become obvious that it’s a field you can type in. 

I would suggest you make the search field obvious using a text field UI element, ideally pre-activated and ready to type in. Below, how about recent searches by the user, then trending searches? You could have a user search below this, if desired.

**Comment by benhalpern** [comment]
Yeah. This is sort of an obvious work in progress. It shouldn't just be a random list of users. This is actually just the default search results if nothing is entered in the bar.

But the pre-activated search bar is sort of weird behavior in this context (trust me we tried it)

The thing this tab is _supposed_ to have are sort of "discover" features. Basically allow you to click into trending parts of the site outside of your regular feed. This is the pattern that Twitter/Instagram etc. have established.

This actual code can currently be fully written from the web's side of things. So the TODO is to get that working properly over in that land.
